src/streams/VirgilStreamCipher.ts

```diff
@@ -19,16 +21,29 @@ export class VirgilStreamCipher extends VirgilStreamCipherBase {
 	 * @param {VirgilPublicKey|VirgilPublicKey[]} publicKeys - A single
 	 * {@link VirgilPublicKey} or an array of {@link VirgilPublicKey}'s to
 	 * to encrypt the data with.
+	 * @param {Data} [signature] - Optionally add a signature of plain data to the encrypted stream.
+	 * @param {StringEncoding} [encoding] - If `signature` is a string,
+	 * specifies its encoding, otherwise is ignored. Default is 'base64'.
 	 */
-	constructor (publicKeys: VirgilPublicKey|VirgilPublicKey[]) {
+	constructor(
+		publicKeys: VirgilPublicKey | VirgilPublicKey[],
+		signature?: Data,
+		encoding: StringEncoding = 'base64'
```

Comment:
How about joining `signature` and `encoding` into a single argument of type `object` with two properties - `signature` and `encoding`?